builder-support/debian/authoritative/debian-buster/tests/supported-algos

```diff
@@ -10,7 +10,7 @@ trap cleanup EXIT
 
 pdnsutil list-algorithms 2>/dev/null | tee "$TMPFILE"
 
-for algo in RSASHA1 RSASHA1-NSEC3-SHA1 RSASHA256 RSASHA512 ECC-GOST ECDSAP256SHA256 ECDSAP384SHA384 ED25519; do
+for algo in RSASHA1 RSASHA1-NSEC3-SHA1 RSASHA256 RSASHA512 ECDSAP256SHA256 ECDSAP384SHA384 ED25519; do
```

Comment:
I _think_ `ED448` (algo 16) is supported via OpenSSL as well.

Reply:
15+16 is a package deal in openssl

Reply:
we either have both or neither

Reply:
added ED448